### ISSUE TITLE
Suggestion: Mega EverDrive: Replace (Rev A) revision of Quackshot with (Rev A) (Alt 1) to allow MiSTer compatibility

### ISSUE THREAD

**Issue by Beatatester** [frederic-mahe/Hardware-Target-Game-Database#686]
The version of Quackshot used in the packs is incompatible with MiSTer, as described in the issue here: https://github.com/MiSTer-devel/Genesis_MiSTer/issues/166

The version currently used in the packs is this:

QuackShot Starring Donald Duck ~ QuackShot - I Love Donald Duck - Guruzia Ou no Hihou (World) (Rev A).md	ca02845fa49cd46ccb0a4023b91b10695573668e	ade4e47b0fbd329acaf36efc4f8d637d	5dd28dd7

The current revision uses a mapper that isn't supported by MiSTer. It also didn't work on a MegaSD when I tried several months ago, though I haven't had the chance to try on the most recent firmware.

According to the issue linked above, there is an alternate version of this revision of the game that does NOT use the problematic mapper:

QuackShot Starring Donald Duck ~ QuackShot - I Love Donald Duck - Guruzia Ou no Hihou (World) (Rev A) (Alt 1)
CRC: A5C946B3

This alternate version is supposedly identical, minus the mapper, and is also in no-intro. Since MiSTer is unlikely to be updated to support the current version of the game any time soon, and since this alternative version is supposedly identical in every way that matters to a user, I believe it would make sense to use this alternative, MiSTer-supported version, instead of the current unsupported version.

**Comment by tjanas** [comment]
What happens when this rom with checksum 5dd28dd7 is used with a Mega EverDrive or MegaSD on real hardware, or when on Mega Sg (either via the flash cart or JB)?

**Comment by Beatatester** [comment]
I don't have access to a Mega Everdrive, but I did try the following:

Mega SG (firmware ver. JB7.7)
Mega SD (firmware 1.04_R09, which is the most recent firmware)
MiSTer (most recent firmware)

And all failed to boot the standard _(Rev A)_ version that is included in the database currently.

The _(Rev A) (Alt 1)_ version works on all three of those devices. 

For further clarity, here are the hashes of the specific ROMs I tested:

![XS6B8ia](https://user-images.githubusercontent.com/53070632/121837918-c19e1e00-cca4-11eb-99ab-9eaa28594fcd.png)
![qML4Eco](https://user-images.githubusercontent.com/53070632/121837919-c236b480-cca4-11eb-98ee-e92533f01dbe.png)
